io/zenoh-transport/tests/unicast_transport.rs

```diff
@@ -535,19 +573,58 @@ async fn run_single(
     .await;
 }
 
-async fn run(
+async fn run_internal(
     client_endpoints: &[EndPoint],
     server_endpoints: &[EndPoint],
     channel: &[Channel],
     msg_size: &[usize],
+    #[cfg(feature = "shared-memory")] shm_transport: bool,
 ) {
     for ch in channel.iter() {
         for ms in msg_size.iter() {
-            run_single(client_endpoints, server_endpoints, *ch, *ms).await;
+            run_single(
+                client_endpoints,
+                server_endpoints,
+                *ch,
+                *ms,
+                #[cfg(feature = "shared-memory")]
+                shm_transport,
+            )
+            .await;
         }
     }
 }
 
+async fn run(
```

Comment:
Let's explicitly call it `run_with_net` in the same way we call `run_with_shm`. This also reflects the actual code organization under `src`.